qualification/conftest.py

```diff
@@ -96,6 +98,7 @@
     IniOption(name="bands", help="Space-separated list of bands to test", type="args", default=["l"]),
     IniOption(name="beams", help="Number of beams to produce", type="string", default="4"),
     IniOption(name="raw_data", help="Include raw data for figures", type="bool", default=False),
+    IniOption(name="array_dir", help="Directory in which to save failed array comparisons", type="paths", default=[]),
```

Comment:
I was wondering why you did `paths = getini("array_dir")` then `path = paths[0]`. The possible types here don't help either.
* Any particular reason you didn't set this as a plain `"string"`?
* Considering we're only expecting/using one (the first) value in this list.

Reply:
With `paths`, pytest will interpret a relative path as relative to the configuration file, even if you invoke pytest from a different directory. That seems like a sensible choice to me. It's annoying that it doesn't have a type for a singleton path.